README.md

```diff
@@ -1,5 +1,9 @@
 # pyramid-learning-journal
 
+<h1>Step 3</h1>
+
+Implemented SQLalchemy and templating on home and detail/post page. Created environmental variables to deploy to Heroku. 
+
 <h1>Step 2</h1>
 
 Step 2 w/ Carlos partnered on the journal server project. Implemented journal entries and incorporated templating to streamline user experience and refactor unnecessary code.
```

Comment:
Good to see you keeping the readme up to date, but try to make it something that describes what your code does, not what you did to your code.